components/console/compass/src/components/Scenarios/ScenarioDetails/ScenarioDetails.js

```diff
@@ -13,12 +13,16 @@ ScenarioDetails.propTypes = {
 
 export default function ScenarioDetails({ scenarioName }) {
   const [applicationsCount, setApplicationsCount] = useState(0);
+  const [runtimesCount, setRuntimesCount] = useState(0);
```

Comment:
looks ok in terms of fix, i was just wondering if for the long term solution, we would prefer to actually allow scenario deletion if there are apps and runtimes in it with a proper confirmation dialog and this would then result in unlabelling the apps and runtimes, updating the label definition as well as asa deletion

Reply:
Maybe it makes sense, let's cross that bridge when we get there, though. There are a lot of considerations in what is right/wrong, which we have to discuss as a team to decide.

But I feel we're still far from such discussions as we haven't gotten too deep yet and haven't felt a need for that, hence the descision won't be that valuable.